js/interface.js

```diff
@@ -61,10 +61,19 @@ var hasAllScreenshots = false;
 var screenshotValidationNotRequired = false;
 var spinner = '<i class="fa fa-spinner fa-pulse fa-fw fa-lg"></i>';
 
+var loginToSocket = true;
 var socket = Fliplet.Socket({
-  login: true
+  login: loginToSocket
 });
 var socketClientId;
+var socketConnected = false;
+
+socket.on(loginToSocket ? 'loginSuccess' : 'connect', function () {
+  socketConnected = true;
```

Comment:
here you should reset `loginToSocket` to `false`, otherwise a reconnection won't update your boolean. In other words, you want to listen for `connected` too.

Reply:
But a reconnection will emit `loginSuccess` too, no? That's what you said on Slack.